quiz.js

```diff
@@ -1 +1,89 @@
-//Your quiz functionality goes here
\ No newline at end of file
+//Your quiz functionality goes here
+document.addEventListener('DOMContentLoaded', e => {
+
+    const startButton = document.getElementById('startBtn');
+    const questionContainer = document.getElementById('questionContainer');
+    const questionElement = document.getElementById('question');
+    const answerButtons = document.getElementById('answerBtns');
+    const nextBtn = document.getElementById('nextBtn');
+    const ansBtn1 = document.getElementById('btn1');
+    const ansBtn2 = document.getElementById('btn2');
+    const ansBtn3 = document.getElementById('btn3');
+    const ansBtn4 = document.getElementById('btn4');
+    const scoreText = document.getElementById('scoreText');
+    const currentScore = document.getElementById('score');
+
+    startButton.addEventListener('click', gameStart)
+
+    let shuffledQ, currentQ, score = 0;
+
+    nextBtn.addEventListener('click', () => {
+        currentQ++;
```

Comment:
You could add some code here to remove the 'correct' class from your buttons at the start of each question, something like `ansBtn1.classList.remove('correct')`